packages/bits/demo/src/components/demo/wizard-v2/wizard-busy/wizard-busy.example.component.html

```diff
@@ -1,6 +1,6 @@
 <div nui-busy [busy]="busy" class="example-container">
     <div class="busy-content">
-        <nui-wizard-horizontal>
+        <nui-wizard-horizontal linear>
```

Comment:
What is this for exactly? Whatever it's doing I wonder if it could be built into the wizard instead of requiring the consumer to do it.

Reply:
Out of curiosity, if it's an input, why doesn't it use notation like this: `[linear]="true"`?

Reply:
Typically, if there is no input specified for the component, the input remains `undefied` (if it wasn't initialized in the first place). But in case the input is specified without a value assigned to it, the input receives an empty string by default. 
Angular Material team treats`""` as a truthy value for their input here with this function:

```
export function coerceBooleanProperty(value: any): boolean {  return value != null &amp;&amp; `${value}` !== 'false';}
```

![image](https://user-images.githubusercontent.com/22597043/122241190-073c3000-cecb-11eb-9fcf-c23e19c8b00d.png)

Reply:
I think I would prefer a more explicit `[linear]="true"` to prevent confusion and also to avoid relying on Material's coersion which could be changed at any time. Additionally, can we set it to true by default in the wizard and have consumers turn it off if they want to override the behavior? It seems like linear would be the more common case.

Reply:
I think this is pretty neat and readable, and explicit enough thanks to their docs. I wouldn't be worrying about material changing things since we're relying on them so heavily if there will be any breaking changes not only this will blow up. Personally, I prefer following the official docs and guides because Material is responsible for the support of their docs and the functionality they describe. As consumers, we don't have to inspect their source code, usually reading the docs is enough. If we deviate from the prescribed instructions we'll be to blame due to the component misuse, even if there is a bug on their side. Just like the empty string may stop working in their method over some time due to the method changes, the directly passed in `true` boolean may stop working either because of the same reason. 

We may have it as a default option though,  I also see this being the most used option.

Reply:
Okay, if it's that way in their docs, that sounds fine to me. And, setting it to linear by default sounds good too. Maybe we could have a link to the Material usage from our docs as well?